docs/data/date-pickers/custom-field/custom-field.md

```diff
@@ -132,6 +132,13 @@ you can replace the field with an `Autocomplete` listing those dates:
 
 {{"demo": "PickerWithAutocompleteField.js", "defaultCodeOpen": false}}
 
+### Using a masked `TextField`
+
+If you want to use a simple mask approach for the field editing instead of the built-in logic,
+you can replace the field with a masked-based `TextField` built using the [rifm](https://www.npmjs.com/package/rifm) package.
```

Comment:
slight variation in wording, since 'masked-based' did sound a bit off.

```suggestion
you can replace the default field with a `TextField` using a masked input value built with the [rifm](https://www.npmjs.com/package/rifm) package.
```